### DISCUSSION TITLE
Default index.html doesnt load any custom javascript other than index.js

### DISCUSSION THREAD

**Discussion by pratikgcet** [codesandbox/sandpack#844] [Q&A]
Here is the sandbox
https://codesandbox.io/s/upbeat-euler-8kx1h1?file=/src/App.tsx
In the sandbox when the browser loads, the test.js is not loaded and is ignored. Only index.js is loaded by default. if loaded the console.log statement in the test.js would have been printed

**Comment by DeMoorJasper** [comment]
Hey, this is a known issue with the sandpack-bundler, it only executes one entrypoint, in this case index.js (which is configured in pkg.json, not read from the html file, sandpack defaults to a default pkg.json)

In case the goal of your project is a bit more codepen like without any bundling/npm modules the upcoming static template might resolve this: https://github.com/codesandbox/sandpack/pull/830

**Reply by pratikgcet** [reply]
Thank you @DeMoorJaspe. Maybe for this scenario I can look to use a different entry point using the entry parameter which should then bundle my custom js?
